hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -245,36 +239,23 @@ static QString* get_var(struct PixelShader *ps, int reg, bool is_dest)
         }
         break;
     case PS_REGISTER_C0:
-        /* TODO: should the final stage really always be unique? */
         if (ps->flags & PS_COMBINERCOUNT_UNIQUE_C0 || ps->cur_stage == 8) {
-            QString *reg = qstring_from_fmt("c_%d_%d", ps->cur_stage, 0);
+            QString *reg = qstring_from_fmt("c0_%d", ps->cur_stage);
```

Comment:
Order got swapped here, is that right? What would have been `c_X_0` is now `c0_X`

Reply:
Yes, that's the intention with this commit.

The register names (used by D3D) are C0 and C1. GL calls them GL_CONSANT_COLOR0_NV and GL_CONSANT_COLOR1_NV.

I think it's weird to call them "C[i]0" and "C[i]1" (which is basically what we do in master).